ruby/vampires.rb

```diff
@@ -0,0 +1,53 @@
+puts "How many employees will be processed today?"
+new_employees = gets.chomp.to_i
+
+def interview
+	is_vampire = nil
+	allergy = nil
+	until allergy == "sunshine" || allergy == "done"
+		puts "Please list any allergies, one at a time.  Type 'done' when finished."
+		allergy = gets.chomp
+	end
+	if allergy == "sunshine"
+		is_vampire = "Probably a vampire."
+	end
+
+	if is_vampire == "Probably a vampire."
+		puts is_vampire
+	else
+		puts "What is your name?"
+		emp_name = gets.chomp
+		puts "How old are you?"
+		age = gets.chomp.to_i
+		puts "What year were you born?"
+		birth_year = gets.chomp.to_i
+		puts "Would you like some garlic bread?"
+		likes_garlic = gets[0] == "y"
+		puts "Do you want health insurance?"
+		wants_insurance = gets[0] == "y"
+
+		age_right = (2016 - age) == birth_year

```

Comment:
Look at the time class in the Ruby docs to fine Time.now.year and see if you can find a dynamic way to add this to your code to make sure that we don't have to rewrite this file every year.